app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,47 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+  def channels
+
+    if params[:uuid] == nil
```

Comment:
IMO we should enforce a `uuid` as a RESTful convention

Reply:
Sure, I'd like to hear more of what you had in mind

Do we want to expose a list of channel_id's at `/some_route` ?

Reply:
Looked into a few API's, I think this is okay as is, it seems common practice to return all of `resource` at `/resource`, and a specific `resource` at `/resource/:id`